spec/requests/planning_application_list_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "API request to list planning applications", type: :request, show_exceptions: true do
+  let(:access_control_allow_origin) { response.headers['Access-Control-Allow-Origin'] }
+  let(:access_control_allow_methods) { response.headers['Access-Control-Allow-Methods'] }
+  let(:access_control_allow_headers) { response.headers['Access-Control-Allow-Headers'] }
+  let(:reviewer) { create :user, :reviewer }
+
+  before do
+    create(:decision, :granted, user: reviewer)
+  end
+
+  describe "format" do
+    it "responds to JSON" do
+      get "/api/v1/planning_applications"
+      expect(response).to be_successful
+    end
+
+    it "sets CORS headers" do
+      get "/api/v1/planning_applications"
+
+      expect(response).to be_successful
+      expect(access_control_allow_origin).to eq('*')
+      expect(access_control_allow_methods).to eq('GET')
+      expect(access_control_allow_headers).to eq('Origin, X-Requested-With, Content-Type, Accept')
+    end
+  end
+
+  describe "data" do
+    let(:data) { json["data"] }
+
+    it "returns an empty response if no petitions are public" do
+      get "/api/v1/planning_applications.json"
+
+      expect(response).to be_successful
+      expect(data).to be_empty
+    end
+
+    it "returns a list of serialized petitions in the expected order" do
```

Comment:
We should update this and the above spec to refer to `planning applications` instead.